lib/locreator.js

```diff
@@ -12,6 +13,16 @@ var Locreator = function (nemo) {
   this.drivex = Drivex(nemo.driver, nemo.wd);
   this.locatex = Locatex(nemo);
   this.normify = this.normalize = function (locator) {
+    if(!locator.locator || !locator.locator.trim()) {
+      error('NO or EMPTY locator found, please fix it');
```

Comment:
I don't remember the conversation, but if the plugin is in an unstable state (as it would be in this case) then I think it's a good idea to create an error and send it via the callback.